task/sonarqube-scanner/0.1/sonarqube-scanner.yaml

```diff
@@ -21,32 +21,25 @@ spec:
   workspaces:
     - name: source-dir
     - name: sonar-settings
+    - name: sonar-secret
```

Comment:
we keep this secret as optional so that it's not necessary to mount in the TaskRun
```yaml
workspaces:
   - name: sonar-secret
     optional: true

Reply:
Oh ok.
Should I do for sonar-settings also cause it is optional too?

Reply:
yes you can but please bump the new version of the task and do changes over there so that the following changes does not affect the current task

Reply:
Sure